archive.php

```diff
@@ -16,9 +16,8 @@
  * @package WordPress
  * @subpackage Twenty_Twelve
  * @since Twenty Twelve 1.0
- 
-
  */
+
```

Comment:
These tags should match those from `author.php` - the `@package` line should read MITLibraries-News. The `@since` line is correct. There should be no `@subpackage` line.